shiny/api-examples/update_popover/app-express.py

```diff
@@ -0,0 +1,32 @@
+from shiny import reactive, req
+from shiny.express import input, ui
+
+ui.input_action_button("btn_show", "Show popover", class_="mt-3 me-3")
+ui.input_action_button("btn_close", "Close popover", class_="mt-3 me-3")
+
+ui.br()
+ui.br()
+
+with ui.popover(id="popover_id"):
+    ui.input_action_button("btn_w_popover", "A button w/ a popover", class_="mt-3")
+    "A message"
+
+
+@reactive.effect
+def _():
+    req(input.btn_show())
+
+    ui.update_popover("popover_id", show=True)
```

Comment:
I made these use `@reactive.event()` when I translated a similar app

```suggestion
@reactive.effect
@reactive.event(input.btn_show)
def _():
    ui.update_popover("popover_id", show=True)
```